src/main/java/kevin/Kevin.java

```diff
@@ -0,0 +1,56 @@
+package kevin;
+
+import kevin.evaluator.Evaluator;
+import kevin.parser.Parser;
+import kevin.parser.FileParser;
+import kevin.parser.QueryObject;
+import kevin.ui.Logger;
+import kevin.exception.KevinException;
+import kevin.storage.FileStorage;
+import kevin.taskList.TaskList;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Kevin class, a task list chatbot.
+ */
+public class Kevin {
+    public static void main(String[] args) {
+        Logger logger = new Logger();
+        Parser parser = new Parser();
+        FileParser fileParser = new FileParser();
+        FileStorage fileStorage = new FileStorage();
+        TaskList taskList = new TaskList();
+        Scanner scanner = new Scanner(System.in);
+        Evaluator evaluator = new Evaluator(logger, fileStorage, taskList);
+
+        logger.hello();
+
+        try {
+            fileStorage.initialize();
+            ArrayList<String> unparsedTasks = fileStorage.getTasksFromFile();
+            for (String s : unparsedTasks) {
+                evaluator.evaluate(fileParser.parseLine(s), true);
+            }
+        } catch (KevinException ke) {
+            logger.log(ke.getMessage());
+        }
+
+
+        while (true) {
```

Comment:
Maybe you could consider whether all these actions be possibly passed into separate functions (e.g. command() / startup() ) to abstract their functions away from each other?